client/src/components/FriendList.js

```diff
@@ -1 +1,11 @@
 import React from "react";
+
+function FriendList() {
+    return (
+      <div className="friendList">

```

Comment:
Can you please change all the class names to kebab case (In this case, `friendList` --> `friend-list`)? (You'll also need to change the CSS selector accordingly)